docs/source/sysadmin_guide/off_chain_settings.rst

```diff
@@ -191,13 +191,13 @@ Additional steps specify the peers for this node, change the scheduler type
 
          .. code-block:: python
 
-            python
+            python3
```

Comment:
The format of the key seems to change when I use python3 and python3-zmq to generate these keys. All the keys I have created with python3 start with a b and are single quoted. Such as `b'se&dCup0nhqi+ku-HNvWg1HuH5I0veFw?!FJ6u/c'`